### PR TITLE
Pull Request 1

### PR DESCRIPTION
**Pull request by flatelen**
Sweet

### CODE REVIEW

**Comment by flatelen** [comment]
Badadeeboom

**Comment by JonLim** [comment]
Hi hello

**Comment by jamesrwu** [comment]
+1

**Comment by skerrwork23** [comment]
yo yo yo

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 Tutorial Repository
 
 Just another commit
+
+This is the pull request I always wanted to make
```
Comment (flatelen) [inline README.md]: Weeeeee test test test

Comment (flatelen) [reply README.md]: dump

**Comment by flatelen** [comment]
Some other stuff

File: README.md (hunk 1 above)
Comment (flatelen) [reply README.md]: mega hockey

**Comment by flatelen** [comment]
TestingThirtyHorses

**Comment by flatelen** [comment]
MegaBombReview

**Pull request merged**